api/app/controllers/concerns/group_admin_concern.rb

```diff
@@ -0,0 +1,18 @@
+module GroupAdminConcern
+  extend ActiveSupport::Concern
+
+  private
+
+  def authorize_group_admin!
+    return true if @group.admin?(current_user)
+
+    Rails.logger.info "User with ID ##{current_user.id} is not this group's Admin"
+    render json: {
+      errors: {
+        group: ["El usuario con ID ##{current_user.id} no es administrador de este grupo"]
+      }
+    }, status: :unauthorized
+
+    false
```

Comment:
Por qué estamos retornando `true/false` acá?

No es suficiente hacer el render del json nomás cuando no está autorizado?

```suggestion
    return if @group.admin?(current_user)

    Rails.logger.info "User with ID ##{current_user.id} is not this group's Admin"
    
    render json: {
      errors: {
        group: ["El usuario con ID ##{current_user.id} no es administrador de este grupo"]
      }
    }, status: :unauthorized
```

Reply:
salado, mas compacto, corregido